compose.yml

```diff
@@ -58,7 +58,7 @@ services:
       # This will only be run the FIRST time the container is started, when
       # it's setting up the default database. Once you've loaded the data and
       # deleted the backup file, comment this line out again.
-      - ./db-dump/the-world_live_2025-09-07T18-00-00_UTC_database.sql:/docker-entrypoint-initdb.d/initial_load.sql
+      - ./db-dump/the-world_live_2025-09-09T21-51-17_UTC_database.sql:/docker-entrypoint-initdb.d/initial_load.sql
```

Comment:
We really need to find a better way to do this so this file doesn't keep getting committed when someone updates their local db and forgets to not commit this file.